Change overall structure (#35)
11michalis11/ED-EMS-game-paper: sections/introduction/main.tex

As a result, ambulance blocking not only impacts on patients waiting for ED
service, but has a major knock-on effect to delaying the ability of ambulances
to respond to new EMS calls, thus placing lives at risk~\cite{eastanglia}.

> There are numerous news articles that focus on the problems that arise when 

Reviewer: ```suggestion
There are numerous news articles that focus on the complexity that arises when 
```

ambulances stay blocked outside of the hospital for a long amount of 
time~\cite{dailyrecords, staffordshirelive}.

> Some news reports also comment on the long idle time of ambulances when 

Reviewer: ```suggestion
Some news reports  comment on the long idle time of ambulances when 
```

in choosing when to start blocking ambulances.
This is similar to~\cite{deo2011centralized}.

% Major contributions

> The novelty of our research contribution in this paper is to consider:

Reviewer: ```suggestion
The major contributions of this paper are:
```

\begin{itemize}
    \item A queueing model with 2 consecutive waiting spaces where one would 
    serve as a parking space for the ambulances

>     \item Performance measure formulas for the queueing model

Reviewer: ```suggestion
    \item Analytic performance measure formulas for the queueing model
```

    \item A 3-player game theoretic model between the EMS and two EDs

>     \item The emergence of gaming behaviour between EDs and the EMS.

Reviewer: ```suggestion
    \item Numerical experiments showing emergent behaviour of gaming between EDs and the EMS.
```

emerge and ways to apply some incentive mechanisms to improve them.
The EDs are modelled as two queueing systems each with a tandem buffer and a 
service centre.
The performance measures are then used as the utilities of the game.
The novelty of the queueing model here is a contribution not only the game 
theoretic literature but also to the queueing theoretic literature.
To the authors knowledge, no such model of a tandem queueing model with a pair 
of parameters for the buffer has been considered.
Although our research is motivated by the particular EMS-ED example, our 
developed modelling framework and behavioural insights has application to 
similar systems across a range of sectors and settings. 

% Sign posting
This paper consists of two main sections.
The first section presents a novel queueing model for a hospital with two
types of patients and two waiting zones.
A detailed description of how to acquire the performance measure formulas of 
such queueing system is given.
The next section gives an overview of the game theoretic model and several
theoretic results pertaining to the performance measures of this model which 

> are used to build the utilities of the game.

Reviewer: Refer to the sections directly?

Reviewer 2: Will do 👍
